prow/gerrit/adapter/adapter.go

```diff
@@ -525,6 +532,32 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 	return nil
 }
 
+// shouldSkipProcessingChange returns true when there is no new commit or relevant commands in the comment messages
+func (c *Controller) shouldSkipProcessingChange(change client.ChangeInfo, lastProjectSyncTime time.Time) bool {
+	if lastProjectSyncTime.IsZero() {
+		return false
+	}
```

Comment:
We don't need this if we just use `syncTime`.

I can see that you'd trying to handle the case when we are processing a repo for the first time ever, but the behavior will already be correct in this case if you just use `syncTime`.  Note that every change that would only be processed due to this new check would ultimately be skipped due to the way we set the last sync time to `Now()` here as well: https://github.com/kubernetes/test-infra/blob/53438db9113bfbc63b477715248341228c6083df/prow/gerrit/adapter/adapter.go#L706C3-L710
(side note about the lines I linked: we could skip the locking and map duplication just for that lookup if you pass `syncTime` down to `triggerJobs()` as well. That would be nice to include since we're now passing that to `processChange`.)